semantic/polytype.go

```diff
@@ -16,6 +16,11 @@ type PolyType struct {
 	fb *fbsemantic.PolyType
 }
 
+// For testing we need an exported method that checks whether PolyType.fb is nil.
+func (pt PolyType) IsNil() bool {
+	return pt.fb == nil
+}
+
```

Comment:
@wolffcm is this the change that you were asking for?

Reply:
Yes, exactly.